src/components/shared/Datatable.vue

```diff
@@ -110,6 +110,18 @@
                     return data
                 },
             },
+
+            /**
+             * getSortParam function for altering vuetables default sort query string
+             *
+             * The default for this prop works with core
+             */
+            getSortParam: {
+                type: Function,
+                default(sortOrder) {
+                    return sortOrder.map(sort => (`${sort.sortField}, ${sort.direction}`))
```

Comment:
This creates a query param that isn't compatible with our API, `?order_by[]=name,+asc`, `order_by` should be a string, not an array. I think adding a join to the end will help here.

```js
return sortOrder.map(sort => (`${sort.sortField}, ${sort.direction}`)).join('')
```

Have you tested this with core?

Reply:
I have updated this method to account for multi-sort if we choose to use it.. it adds a pipe between each sort field is that ok or would you prefer a different separator? this also prevents it being parsed as an array

Reply:
Pipe is fine.